src/components/destination_list.js

```diff
@@ -2,32 +2,38 @@ import React from "react";
 import PropTypes from "prop-types";
 import { List } from "react-virtualized/dist/commonjs/List";
 
-import Column from "./column/column";
 import ItemsList from "./list/items_list";
 import NoItems from "./items/no_items";
 import SelectedItem from "./items/selected_item";
 import SelectionStatus from "./selection_status/selection_status";
 import { groupItems } from "./item_grouping_util";
+import withSearch from "./with_search";
 
 const DestinationList = ({
   selectionStatusRenderer,
   selectedIds,
   clearAll,
   messages,
-  selectedItems,
   itemHeight,
   height,
   unselectItems,
   selectedItemRenderer,
   noItemsRenderer,
-  withGrouping
+  withGrouping,
+  filteredItems,
+  showSearch,
+  selectedItems
 }) => {
   const SelectionStatusRenderer = selectionStatusRenderer;
+
+  const filteredSelectedItemsList =
+    showSearch || filteredItems ? filteredItems : selectedItems;
   const updatedSelectedItems = withGrouping
-    ? groupItems(selectedItems)
-    : selectedItems;
+    ? groupItems(filteredSelectedItemsList)
+    : filteredSelectedItemsList;
+
   return (
-    <Column>
+    <>
```

Comment:
I would prefer to [return array](https://egghead.io/lessons/react-render-multiple-elements-without-a-wrapping-element-in-a-component-in-react-16) instead of wrapping in something like this.
unless you had good reason for it

Reply:
i mean, isn't it supported since react 16.2 ([link](https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html))?..